docs/diginstroom/sip/1.0/profiles/basic.md

```diff
@@ -68,7 +68,7 @@ root_directory
 - The `dc*.xml` file MUST adhere to the restrictions on cardinality of terms outlined in the table below; if a term is not listed with a restriction on cardinality, it MAY be used multiple times.
 - The `dc*.xml` file MUST contain a shared ID with a PREMIS object in the `preservation/premis.xml` file, stored in the `<dcterms:identifier>` element (see [next section](#connecting-the-descriptive-metadata-to-premis)).
 - The `dc*.xml` file MUST NOT contain additional IDs besides the shared ID in the `<dcterms:identifier>`; these MUST be added in the `preservation/premis.xml` file.
-- A descriptive metadata element of datatype [String]({{ site.baseurl }}{% link docs/diginstroom/sip/1.0/2_terminology.md %}#string) MAY contain an attribute `@xml:lang` that indicates the language of the metadata element's value (in order to, for example, specify a title or description in multiple languages). The value of this attribute MUST be a valid [IETF BCP 47 language tag](https://www.rfc-editor.org/info/bcp47)(see [here](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry) for a list).
+- A descriptive metadata element of datatype [String]({{ site.baseurl }}{% link docs/diginstroom/sip/1.0/2_terminology.md %}#string) MAY contain an attribute `@xml:lang` that indicates the language of the metadata element's value (in order to, for example, specify a title or description in multiple languages). The value of this attribute MUST be a valid [IETF BCP 47 language tag](https://www.rfc-editor.org/info/bcp47)(see [here](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry) for a list). If the attribute is not present, the value is assumed to be in Dutch and the attribute `@xml:lang` with value `nl` is applied automatically.
```

Comment:
Zouden we dit gewoon niet verplichten dat er voor de specifieke nodes altijd een `@xml:lang` attribuut is? Dat maakt het eenvoudiger aan onze kant.

Reply:
Valt iets voor te zeggen. Krijgen we dit verkocht aan de makers van SIPs? Het is alleszins duidelijker